src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -185,13 +185,13 @@ public Task StoreCredentialAsync(InputArguments input)
             _context.Trace.WriteLine("Credential was successfully stored.");
 
             Uri targetUri = GetTargetUri(input);
-            if(IsBitbucketServer(targetUri))
+            if (IsBitbucketServer(targetUri))
             {
                 // BBS doesn't usually include the username in the urls which means they aren't included in the GET call, 
                 // which means if we store only with the username the credentials are never found again ...
                 // This does have the potential to overwrite itself for different BbS accounts, 
                 // but typically BbS doesn't encourage multiple user accounts
-                string bbsCredentialKey = GetBbSCredentialKey(input);
+                string bbsCredentialKey = GetBitbuckerServerCredentialKey(input);
```

Comment:
```suggestion
                string bbsCredentialKey = GetBitbucketServerCredentialKey(input);
```

Whoops!!! My bad sorry!

Reply:
No problem I missed it too, fixed it locally.
It is an interesting gotcha when using the suggested commits, I'm not sure how I feel about them yet.

Reply:
Yeah. I feel they're a good idea, but think they need some work. Perhaps starting with being able to group suggestions into one committable set.